daprdocs/content/en/developing-applications/building-blocks/actors/howto-actors.md

```diff
@@ -129,3 +129,16 @@ DELETE http://localhost:3500/v1.0/actors/<actorType>/<actorId>/reminders/<name>
 ```
 
 Refer [api spec]({{< ref "actors_api.md#invoke-reminder" >}}) for more details.
+
+## Actor Runtime Configuration
+
+You can configure the Dapr Actors runtime configuration to modify the default runtime behavior.
+
+### Current defaults
```

Comment:
```suggestion
### Configuration parameters
```

I'd generalize this a little bit because the items show both the parameter definitions and defaults